src/Plugins/Nop.Plugin.Widgets.NivoSlider/Controllers/WidgetsNivoSliderController.cs

```diff
@@ -48,18 +48,23 @@ public IActionResult Configure()
                 Picture1Id = nivoSliderSettings.Picture1Id,
                 Text1 = nivoSliderSettings.Text1,
                 Link1 = nivoSliderSettings.Link1,
+                AltText1 = nivoSliderSettings.AltText1,

```

Comment:
Formatting issue: you need to add an extra line between blocks as it was done before.